src/couch/src/couch_db.erl

```diff
@@ -736,19 +736,23 @@ validate_security_object(SecProps) ->
 
 % validate user input
 validate_names_and_roles({Props}) when is_list(Props) ->
-    case couch_util:get_value(<<"names">>,Props,[]) of
+    case couch_util:get_value(<<"names">>, Props, []) of
     Ns when is_list(Ns) ->
             [throw("names must be a JSON list of strings") ||N <- Ns, not is_binary(N)],
             Ns;
-    _ -> throw("names must be a JSON list of strings")
+    _ ->
+        throw("names field vallue must be a list of strings")
```

Comment:
typo in "vallue".

why change of error message, btw? I think it's clear as it is

Reply:
here if request body is like 
`'{"members":{"names": "user1"}}'`
the logical will enter into line 744, throw an exception, but here key of `names` existed just value is not correct, the value should like this `["user1"]`, so I update the description.
but you think this is clear `names must be a JSON list of strings`, I can change it back.
to be honest I don't quiet know what it  a `JSON list of strings` look like.

Reply:
JSON list or array is an ordered list of zero or more values represented by square brackets. JSON string is a sequence of zero or more unicode characters delimited with double quotes.

So valid JSON list of strings is something that looks like `[]` or `["user1"]` or `["user1", "user2"]`. Since a security object has to be represent as a valid JSON, I think it's appropriate to refer to JSON when security object's content is invalid.